packages/api/amplify_api/example/android/app/src/main/kotlin/com/amazonaws/amplify/example/MainActivity.kt

```diff
@@ -0,0 +1,6 @@
+package com.amazonaws.amplify.example
+
+import io.flutter.embedding.android.FlutterActivity
```

Comment:
Double check this. There should already be a MainActivity, but the package might have become out of sync